scripts/cxg_remaster.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+import time
+
+import numpy as np
+import psutil
+import tiledb
+
+"""
+Re-master an existing "X" tiledb array in a CXG
+Creates a new array with the same data but a different tdb schema
+"""
+
+X_extent = None
+X_name = None
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("cxg", type=str, help="cxg path")
+    parser.add_argument("--kind", choices=["sparse", "dense"], type=str, help="target: sparse or dense")
+    parser.add_argument("--obs", type=int, help="obs extent", default=512)
+    parser.add_argument("--var", type=int, help="var extent", default=2048)
+    parser.add_argument("--cell-order", choices=["row", "col"], type=str, default="row")
+    parser.add_argument("--tile-order", choices=["row", "col"], type=str, default="col")
+    parser.add_argument("--capacity", type=int, default=128000)
+    parser.add_argument("--compression", type=int, default=22)
+    parser.add_argument("--name", type=str, default="X_new")
```

Comment:
Nit: it might add clarity if the two array names were clearly "source" and "target", or something like that.

eg, `--src-array`, `--dst-array`